apps/challenges/views.py

```diff
@@ -147,8 +147,8 @@ def add_participant_team_to_challenge(request, challenge_pk, participant_team_pk
 @api_view(['POST'])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
 @authentication_classes((ExpiringTokenAuthentication,))
-def disable_challenge(request, pk):
-
+@permission_classes((IsChallengeCreator,))
+def disable_challenge(request, challenge_pk):
```

Comment:
Done. Sorry about that. I assumed a consistent naming scheme would make life easier.